// @sicaho-collab/design-tokens/dist/index.cjs
'use strict';

// src/colors.ts
var m3Colors = {
  // Primary
  primary: "#9A76BE",
  onPrimary: "#FFFFFF",
  primaryContainer: "#EADDFF",
  onPrimaryContainer: "#21005D",
  // Secondary
  secondary: "#625B71",
  onSecondary: "#FFFFFF",
  secondaryContainer: "#E8DEF8",
  onSecondaryContainer: "#1D192B",
  // Tertiary
  tertiary: "#7D5260",
  onTertiary: "#FFFFFF",
  tertiaryContainer: "#FFD8E4",
  onTertiaryContainer: "#31111D",
  // Error
  error: "#B3261E",
  onError: "#FFFFFF",
  errorContainer: "#F9DEDC",
  onErrorContainer: "#410E0B",
  // Surface
  surface: "#FFFFFF",
  surfaceDim: "#D9D9D9",
  surfaceBright: "#FFFFFF",
  surfaceContainerLowest: "#FFFFFF",
  surfaceContainerLow: "#F7F7F7",
  surfaceContainer: "#F1F1F1",
  surfaceContainerHigh: "#EBEBEB",
  surfaceContainerHighest: "#E3E3E3",
  onSurface: "#1D1B20",
  onSurfaceVariant: "#49454F",
  outline: "#7A7A7A",
  outlineVariant: "#C9C9C9",
  inverseSurface: "#313131",
  inverseOnSurface: "#F4F4F4",
  inversePrimary: "#D0BCFF",
  scrim: "#000000",
  shadow: "#000000"
};

// src/shadows.ts
var m3Shadows = {
  1: "0px 1px 2px rgba(0, 0, 0, 0.3), 0px 1px 3px 1px rgba(0, 0, 0, 0.15)",
  2: "0px 1px 2px rgba(0, 0, 0, 0.3), 0px 2px 6px 2px rgba(0, 0, 0, 0.15)",
  3: "0px 4px 8px 3px rgba(0, 0, 0, 0.15), 0px 1px 3px rgba(0, 0, 0, 0.3)",
  4: "0px 6px 10px 4px rgba(0, 0, 0, 0.15), 0px 2px 3px rgba(0, 0, 0, 0.3)",
  5: "0px 8px 12px 6px rgba(0, 0, 0, 0.15), 0px 4px 4px rgba(0, 0, 0, 0.3)"
};

// src/radius.ts
var m3Radius = {
  none: 0,
  xs: 4,
  sm: 8,
  md: 12,
  lg: 16,
  xl: 28,
  full: 9999
};

// src/spacing.ts
var m3Spacing = {
  0: 0,
  1: 4,
  2: 8,
  3: 12,
  4: 16,
  5: 20,
  6: 24,
  8: 32,
  10: 40,
  12: 48,
  16: 64
};

// src/typography.ts
var m3Typography = {
  fontSans: "'Instrument Sans', system-ui, -apple-system, sans-serif"
};

exports.m3Colors = m3Colors;
exports.m3Radius = m3Radius;
exports.m3Shadows = m3Shadows;
exports.m3Spacing = m3Spacing;
exports.m3Typography = m3Typography;
